tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -84,6 +84,14 @@ mount.gcsfuse $TEST_BUCKET_NAME $MOUNT_DIR -o only_dir=testDir,implicit_dirs=tru
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/operations/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR
 sudo umount $MOUNT_DIR
 
+# Run tests with config "create-empty-file: true".
+echo "write:
```

Comment:
Why specifically these tests when create-empty-file:true?

Reply:
This is the script that GKE team uses to run tests. They asked us to provide command to run tests with "create-empty-file: true" config. 

Ref: https://docs.google.com/document/d/1O5bs74S9AdIrHTVunczQ29avrGK8UDZBIjwxIa4UFa0/edit#bookmark=id.hpstc8rn3wxl